File: Assignment__33/Question2.java
import java.util.*;

class Alphabet
{
    public boolean Check(char chh)
    {
        if((chh >= 'A' && chh <= 'Z'))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}
class Question2
{
    public static void main(String A[])
    {
        boolean bRet = false;

        Scanner scanobj = new Scanner(System.in);
        System.out.println("Enter the Character : ");

        char ch = scanobj.next().charAt(0);               //Need to remember

        Alphabet aobj = new Alphabet();

        bRet = aobj.Check(ch);

        if(bRet == true)
        {
            System.out.println("It is a Capital Alphabet.");
        }
        else
        {
            System.out.println("It is not a Capital Alphabet.");
        }

    }
}

File: Assignment__33/Question2.c
#include<stdio.h>
#include<stdbool.h>

bool check(char *str)
{
    while(*str != '\0')
    {
        if((*str >= 'A' && *str <= 'Z'))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

int main()
{
    char Arr[50] = "\0";
    bool bRet = false;

    printf("Enter the String : \n");
    scanf("%[^\n]s",Arr);

    bRet = check(Arr);

    if(bRet == true)
    {
        printf("It is a Capital Alphabet.");
    }
    else
    {
        printf("It is not a Capital Alphabet");
    }

    return 0;
}